src/identity-handler.js

```diff
@@ -33,11 +40,17 @@ IdentityHandler.prototype.onLoginComplete = function(
 IdentityHandler.prototype.onLogoutComplete = function(
     mParticleUser,
     identityApiRequest
-) {};
+) {
+    window.heap.resetIdentity();
+};
 IdentityHandler.prototype.onModifyComplete = function(
     mParticleUser,
     identityApiRequest
-) {};
+) {
+    if (forwarderSettings.userIdentificationType == type) {
```

Comment:
I think I saw that you assigned `userIdentificationType` to common, so I think you can use that instead.

Also, use `===` for JS for more accurate comparisons.

```suggestion
    if (common.userIdentificationType === type) {
```

Reply:
Updated